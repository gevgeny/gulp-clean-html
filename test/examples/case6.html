<div>
    <div my-attr="foo">
        node1
    </div>
    <div my-attr="bar">
        node2
    </div>
    <div my-attr="baz">
        node3
    </div>
    <div>
        node4
    </div>
</div>
<div my-attr="foo">
    node5
</div>
<div my-attr="bar">
    node6
</div>
<div my-attr="baz">
    node7
</div>
