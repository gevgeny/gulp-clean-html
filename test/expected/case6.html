<div>
    <div my-attr="foo">
        node1
    </div>
    
    
    <div>
        node4
    </div>
</div>
<div my-attr="foo">
    node5
</div>


